components/03-modules/gallery/gallery.config.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  title: 'Product grid item',
+  context: {
+    class: '',
+    productLabel: {
+      text: 'NEW',
+      type: 'new'
+    },
+    typeHorizontal: false,
+    imageMain: {
+      src: '/images/gallery/product-img-496-496.jpg'
+    },
+    thumbs: [
+      {
+        imageThumb: {
+          src: '/images/gallery/product-img-80-80.jpg',
+          active: true
+        }
+      },
+      {
+        imageThumb: {
+          src: '/images/gallery/product-img-80-80.jpg'
+        }
+      },
+      {
+        imageThumb: {
+          src: '/images/gallery/product-img-80-80.jpg'
+        }
+      },
+      {
+        imageThumb: {
+          src: '/images/gallery/product-img-80-80.jpg'
+        }
+      }
+    ]
+  },
+  variants: [
+    {
+      name: 'horizontal',
+      context: {
+        typeHorizontal: true
+      },
+    },
+    {
+      name: 'with-label',
```

Comment:
Not necessary, we have `if` statement here and when someone want to add badge here will do it. To delete

Reply:
Why it's still there? 😄

Reply:
because I think it can be as variant for easy preview

Reply:
so just add it to the default preview. If someone doesn't want badge won't to fill config with it b/c you have badge in `if`

Reply:
I will leave like disscussed on the meeting